modules/style.js

```diff
@@ -16,7 +16,7 @@ function updateStyle(oldVnode, vnode) {
   var oldHasDel = 'delayed' in oldStyle;
 
   for (name in oldStyle) {
-    if (!style[name]) {
+    if (style[name] === undefined) {
```

Comment:
I don't think that we really expect intentionally falsy values here, like, `false` or `null`, right?